pkg/webservice/handlers.go

```diff
@@ -68,10 +68,12 @@ const (
 	AppStateCompleted = "completed"
 )
 
-var allowedActiveStatusMsg string
-var allowedAppActiveStatuses map[string]bool
-var streamingLimiter *StreamingLimiter
-var maxRESTResponseSize atomic.Uint64
+var (
```

Comment:
I don't have any problem with these changes, but keep in mind that if we make a lot of mostly-cosmetic changes (changing decls from using `var` to `:=`, grouping declarations, removing trailing spaces, etc)  to yunikorn-core, we may have a very tough time getting much (if any) of this merged back upstream someday.

Reply:
I have gofumpt turned on... I'll fix formatting 
Good call!